### ISSUE TITLE
DMP client folder-path checking conflicts with linux symlinks

### ISSUE THREAD

**Issue by kolAflash** [godarklight/DarkMultiPlayer#326]
When starting KSP, DMP gives me a notification window telling me:

```
Incorrect Install Detected:
DarkMultiPlayer is not correctly installed.

Current location:
/home/user/other-folder/Kerbal Space Program/GameData/DarkMultiPlayer/Plugins/DarkMultiPlayer.dll

Correct location:
/home/user/opt/Steam/SteamApps/common/Kerbal Space Program/GameData/DarkMultiPlayer/Plugins/DarkMultiPlayer.dll
```

I use a symlink to link my GameData to from
```/home/user/opt/Steam/SteamApps/common/Kerbal Space Program/GameData ```
to
```/home/user/other-folder/KSP-GameData ```
This is for some organizational stuff in my file-system and I'd like not to change this.

Any other component of KSP is working fine with this. Except DMP. It's always giving me the warning and it's not loading further it the symlink is in use.

Guess in "Client/Utilities/InstallChecker.cs" something needs to be changed.
Alternatively to write some probably complicated code to check if there are symlinks, what about a config-file flag to disable the IsCorrectlyInstalled function by letting it always return true.

**Comment by godarklight** [comment]
You can safely let it return true, In debug builds the check is simply disabled as I do the same thing, run symlinks on linux :).

This is the 1.0 build with the check disabled: http://godarklight.info.tm/symlink/DarkMultiPlayer.dll

I'm not really willing to disable this for everyone as it hits me so often without this check (I package DMP a little differently, The reasoning being I hate when tar -xvf filename.tar.gz sprals crap all over my home folder :P)

**Comment by kolAflash** [comment]
What about a config-flag or an environment variable?

**Comment by godarklight** [comment]
Fixed via a5d3048faf06c9207f28247b024b0ece6b1ebf26
